orbitale/easyimpress-bundle/1.0/config/presentations/home.yaml

```diff
@@ -0,0 +1,16 @@
+# Documentation for yml reference options:
+# https://github.com/Orbitale/EasyImpressBundle#yml-reference
```

Comment:
Can you fix this comment so that it looks like similar ones for other packages?

Reply:
Fixed